src/templates/partials/test-button.html

```diff
@@ -0,0 +1,7 @@
+<button hx-post="/clicked"
```

Comment:
i take it this is to be the download button? we'll need to update this with the actual endpoint but we'll get there :)

Reply:
Yup, exactly right!